### PR TITLE
Add more information to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by dinatale2**
The point of this PR is to flesh out the `CONTRIBUTING.md` so it can serve as a good example/baseline for other LLNL open source projects.

I also moved `LICENSE` to `LICENSE.md`.

### CODE REVIEW

**Comment by dinatale2** [comment]
@IanLee1521 I can't tag you for review. So I'm pinging you this way.

**Comment by davidbeckingsale** [comment]
I'm not sure that the contributing guide is general enough to be usable across lots of projects. Perhaps it would be better to provide a couple of examples files that projects could build on?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,10 +6,92 @@ ask — or submit your issue or pull request anyway. The worst that can happen i
...
+[How To Contribute](#how-to-contribute)
+  * [Reporting an Issue & Feature Suggestions](#reporting-an-issue--feature-suggestions)
+  * [Pull Requests](#pull-requests)
+  * [License](#license)
+  * [Contributor's Declaration](#contributors-declaration)
+
+## How To Contribute
+
+### Reporting an Issue & Feature Suggestions
+You can find a list of all outstanding issues and feature requests on our Github
```
Comment (IanLee1521) [inline CONTRIBUTING.md:23]: `Github` -> `GitHub`

Comment (IanLee1521) [reply CONTRIBUTING.md:23]: Personally, I'm a fan of newlines after a heading in Markdown.

**Comment by IanLee1521** [comment]
I'm leaning towards agreeing with @davidbeckingsale, I wonder if this might be an *example* of a more rigorous project, but I don't think all (or even most) projects will be this rigorous (nor need to be). 

Do you think it would make sense to form this one more generically, in a way that would be applicable to all of our open source projects?

**Comment by dinatale2** [comment]
@IanLee1521 @davidbeckingsale I can provide this one as a sample and we could start including guides that point to various resources (i.e. the contributor's declaration and what it's used for, CI tools, etc) perhaps?

**Pull request closed without merging**